README.md

```diff
@@ -1,127 +1,116 @@
-Where to Find the Code
-======================
-To submit issues and patches please visit https://github.com/DataDog/chef-datadog.
-The code is licensed under the Apache License 2.0 (see  LICENSE for details).
+# Datadog Chef Cookbook
 
-[![Chef cookbook](https://img.shields.io/cookbook/v/datadog.svg?style=flat)](https://github.com/DataDog/chef-datadog)
```

Comment:
Why remove the badges?

Reply:
These were moved to CONTRIBUTING.md so they don't show on the docs site.